languages/csharp/bitwardenSdk.csproj

```diff
@@ -3,39 +3,69 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
```

Comment:
Output isn't an exe though.

Reply:
If I remove this (output type is lib), becouse we have Program.cs I get "Program using top-level statements must be an executable." 
We can leave it as is or remove  Program.cs, what do you want?

Reply:
We should probably make another csproj for the examples. We don't want that bundled with the library.

Reply:
I agree, I have examples in readme file, I think that should be enough?

Reply:
I would prefer moving Program.cs into either a test or example project, and create a solution for the sdk. Otherwise we won't have a way to test everything. `Program.cs` is essentially an integration test as it's written currently.

Reply:
refactored , now we have lib and exe separated